test.ts

```diff
@@ -86,43 +87,39 @@ function trimEmptyLines(str: string, eliminateAll = false): string {
 @suite("typescript", slow(5000), timeout(15000))
 class SuiteTest {
 
-    @params({ target: "es5", ts: "test.suite" })
-    @params({ target: "es6", ts: "test.suite" })
-    @params({ target: "es5", ts: "test.v2.suite" })
-    @params({ target: "es6", ts: "test.v2.suite" })
-    @params({ target: "es5", ts: "only.v2.suite" })
-    @params({ target: "es6", ts: "only.v2.suite" })
-    @params({ target: "es5", ts: "pending.v2.suite" })
-    @params({ target: "es6", ts: "pending.v2.suite" })
-    @params({ target: "es5", ts: "only.suite" })
-    @params({ target: "es6", ts: "only.suite" })
-    @params({ target: "es5", ts: "pending.suite" })
-    @params({ target: "es6", ts: "pending.suite" })
-    @params({ target: "es5", ts: "retries.suite" })
-    @params({ target: "es6", ts: "retries.suite" })
-    @params({ target: "es6", ts: "context.suite" })
+    // @params({ target: "es5", ts: "test.suite" })
```

Comment:
Are these tests supposed to be commented?

Reply:
oh noes...